scripts/lib.sh

```diff
@@ -0,0 +1,59 @@
+function install_yq() {
+	GOPATH=${GOPATH:-${HOME}/go}
+	local yq_path="${GOPATH}/bin/yq"
+	local yq_pkg="github.com/mikefarah/yq"
+	[ -x  "${GOPATH}/bin/yq" ] && return
+
+	case "$(arch)" in
```

Comment:
I wonder if we should be using https://github.com/kata-containers/tests/blob/master/.ci/kata-arch.sh here?

Reply:
done